Conduit/Conduit/Program.cs

```diff
@@ -6,6 +17,33 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
+builder.Services.AddDbContext<ConduitContext>(options => {
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"));
+    // options.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);
+});
+builder.Services.AddScoped<IAuthService,AuthService>();
+
+builder.Services.AddScoped<IpasswordHasher,BycryptPasswordHasher>();
+
+builder.Services.AddTransient<IUserRepositry,UserRepositry>();
+builder.Services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
+builder.Services.Configure<Jwt>(builder.Configuration.GetSection("JWT"));
+
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
```

Comment:
hmmmm, why do I see these changes here? if you checkout the branch from the main so I shouldn't see the changes here,and you can isolate it with DB changes.